simulation_ws/src/sagemaker_rl_agent/markov/environments/object_tracker_env.py

```diff
@@ -63,14 +63,16 @@ def __init__(self):
                                             shape=(screen_height, screen_width, 3), dtype=np.uint8)
 
         #ROS initialization
-        self.ack_publisher = rospy.Publisher('/cmd_vel', Twist, queue_size=100)
-        self.gazebo_model_state_service = rospy.ServiceProxy('/gazebo/set_model_state', SetModelState)
-        rospy.init_node('rl_coach', anonymous=True)
+        rclpy.init()
+        self.node = rclpy.create_node('rl_coach')
+
+        self.ack_publisher = self.node.create_publisher(Twist, '/cmd_vel', 10)
```

Comment:
Why did you change queue size from 100 to 10?

Reply:
Apologize for the inconvenience, queue size should be 100, Hence We will change it for the same.